### PR TITLE
Updated the documentation for CSI Ephemeral Volumes in powerscale.

### PR DESCRIPTION
**Pull request by chimanjain**
# Description
Updated the documentation for CSI Ephemeral Volumes in powerscale.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/743 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/v1/csidriver/features/powerscale.md [hunk 1]
```diff
@@ -341,30 +344,35 @@ spec:
 
 This manifest creates a pod in a given cluster and attaches a newly created ephemeral inline CSI volume to it.
 
+**Note**: Storage class is not supported in CSI ephemeral inline volumes and all parameters are driver specific.
+CSI ephemeral volumes allow users to provide volumeAttributes directly to the CSI driver as part of the Pod spec.
+Following `volumeAttributes` are supported: size, ClusterName, AccessZone, IsiPath, IsiVolumePathPermissions, AzServiceIP.
```
Comment (rsedlock1958) [inline content/v1/csidriver/features/powerscale.md:349]: Change "Following" to "These"

File: content/v1/csidriver/features/powerscale.md [hunk 2]
```diff
@@ -341,30 +344,35 @@ spec:
...
+For reference, check the description of parameters in the following example: [isilon.yaml](https://github.com/dell/csi-powerscale/blob/main/samples/storageclass/isilon.yaml)
+
 ## Topology
+
 ### Topology Support
 
-The CSI PowerScale driver version 1.4.0 and later supports Topology by default which forces volumes to be placed on worker nodes that have connectivity to the backend storage, as a result of which the nodes which have access to PowerScale Array are appropriately labeled. The driver leverages these labels to ensure that the driver components (controller, node) are spawned only on nodes wherein these labels exist. 
+The CSI PowerScale driver version 1.4.0 and later supports Topology by default which forces volumes to be placed on worker nodes that have connectivity to the backend storage, as a result of which the nodes which have access to PowerScale Array are appropriately labeled. The driver leverages these labels to ensure that the driver components (controller, node) are spawned only on nodes wherein these labels exist.
```
Comment (rsedlock1958) [inline content/v1/csidriver/features/powerscale.md:356]: Remove comma after "storage" and add a period (hard stop). Change "as a result of which" to "This results in". Change "are" to "being"

File: content/v1/csidriver/features/powerscale.md [hunk 3]
```diff
@@ -440,7 +449,7 @@ The user can also set the volume limit for all the nodes in the cluster by speci
 
 ## Node selector in helm template
 
-Now user can define in which worker node, the CSI node pod daemonset can run (just like any other pod in Kubernetes world.)For more information, refer to https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector 
+Now user can define in which worker node, the CSI node pod daemonset can run (just like any other pod in Kubernetes world.)For more information, refer to https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
```
Comment (rsedlock1958) [inline content/v1/csidriver/features/powerscale.md:452]: Change "user" to "users". Remove comma after node. Place period outside of parentheses. Insert space before "For"

File: content/v1/csidriver/features/powerscale.md [hunk 4]
```diff
@@ -533,14 +545,14 @@ In the case of operator-based installation, default permission for powerscale di
...
 ## PV/PVC Metrics
 
-CSI Driver for Dell PowerScale 2.1.0 and above supports volume health monitoring. This allows Kubernetes to report on the condition, status and usage of the underlying volumes. 
+CSI Driver for Dell PowerScale 2.1.0 and above supports volume health monitoring. This allows Kubernetes to report on the condition, status and usage of the underlying volumes.
 For example, if a volume were to be deleted from the array, or unmounted outside of Kubernetes, Kubernetes will now report these abnormal conditions as events.
 
 ### This feature can be enabled
-1. For controller plugin, by setting attribute `controller.healthMonitor.enabled` to `true` in `values.yaml` file. Also health monitoring interval can be changed through attribute `controller.healthMonitor.interval` in `values.yaml` file.   
+
+1. For controller plugin, by setting attribute `controller.healthMonitor.enabled` to `true` in `values.yaml` file. Also health monitoring interval can be changed through attribute `controller.healthMonitor.interval` in `values.yaml` file.
```
Comment (rsedlock1958) [inline content/v1/csidriver/features/powerscale.md:555]: The first clause is not a complete sentence.

File: content/v2/csidriver/features/powerscale.md [hunk 5]
```diff
@@ -341,30 +344,35 @@ spec:
 
 This manifest creates a pod in a given cluster and attaches a newly created ephemeral inline CSI volume to it.
 
+**Note**: Storage class is not supported in CSI ephemeral inline volumes and all parameters are driver specific.
+CSI ephemeral volumes allow users to provide volumeAttributes directly to the CSI driver as part of the Pod spec.
+Following `volumeAttributes` are supported: size, ClusterName, AccessZone, IsiPath, IsiVolumePathPermissions, AzServiceIP.
```
Comment (rsedlock1958) [inline content/v2/csidriver/features/powerscale.md:349]: Change "Following" to "These"

File: content/v2/csidriver/features/powerscale.md [hunk 6]
```diff
@@ -341,30 +344,35 @@ spec:
...
+For reference, check the description of parameters in the following example: [isilon.yaml](https://github.com/dell/csi-powerscale/blob/main/samples/storageclass/isilon.yaml)
+
 ## Topology
+
 ### Topology Support
 
-The CSI PowerScale driver version 1.4.0 and later supports Topology by default which forces volumes to be placed on worker nodes that have connectivity to the backend storage, as a result of which the nodes which have access to PowerScale Array are appropriately labeled. The driver leverages these labels to ensure that the driver components (controller, node) are spawned only on nodes wherein these labels exist. 
+The CSI PowerScale driver version 1.4.0 and later supports Topology by default which forces volumes to be placed on worker nodes that have connectivity to the backend storage, as a result of which the nodes which have access to PowerScale Array are appropriately labeled. The driver leverages these labels to ensure that the driver components (controller, node) are spawned only on nodes wherein these labels exist.
```
Comment (rsedlock1958) [inline content/v2/csidriver/features/powerscale.md:356]: Remove comma after "storage" and add a period (hard stop). Change "as a result of which" to "This results in". Change "are" to "being"

File: content/v2/csidriver/features/powerscale.md [hunk 7]
```diff
@@ -440,7 +449,7 @@ The user can also set the volume limit for all the nodes in the cluster by speci
 
 ## Node selector in helm template
 
-Now user can define in which worker node, the CSI node pod daemonset can run (just like any other pod in Kubernetes world.)For more information, refer to https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector 
+Now user can define in which worker node, the CSI node pod daemonset can run (just like any other pod in Kubernetes world.)For more information, refer to https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
```
Comment (rsedlock1958) [inline content/v2/csidriver/features/powerscale.md:452]: Change "user" to "users". Remove comma after "node" and insert a period (full stop). INitial caps "The". Remove parentheses.

File: content/v2/csidriver/features/powerscale.md [hunk 8]
```diff
@@ -533,14 +545,14 @@ In the case of operator-based installation, default permission for powerscale di
...
 ## PV/PVC Metrics
 
-CSI Driver for Dell PowerScale 2.1.0 and above supports volume health monitoring. This allows Kubernetes to report on the condition, status and usage of the underlying volumes. 
+CSI Driver for Dell PowerScale 2.1.0 and above supports volume health monitoring. This allows Kubernetes to report on the condition, status and usage of the underlying volumes.
 For example, if a volume were to be deleted from the array, or unmounted outside of Kubernetes, Kubernetes will now report these abnormal conditions as events.
 
 ### This feature can be enabled
-1. For controller plugin, by setting attribute `controller.healthMonitor.enabled` to `true` in `values.yaml` file. Also health monitoring interval can be changed through attribute `controller.healthMonitor.interval` in `values.yaml` file.   
+
+1. For controller plugin, by setting attribute `controller.healthMonitor.enabled` to `true` in `values.yaml` file. Also health monitoring interval can be changed through attribute `controller.healthMonitor.interval` in `values.yaml` file.
```
Comment (rsedlock1958) [inline content/v2/csidriver/features/powerscale.md:555]: First clause is not a complete sentence.

File: content/v3/csidriver/features/powerscale.md [hunk 9]
```diff
@@ -341,30 +343,35 @@ spec:
 
 This manifest creates a pod in a given cluster and attaches a newly created ephemeral inline CSI volume to it.
 
+**Note**: Storage class is not supported in CSI ephemeral inline volumes and all parameters are driver specific.
+CSI ephemeral volumes allow users to provide volumeAttributes directly to the CSI driver as part of the Pod spec.
+Following `volumeAttributes` are supported: size, ClusterName, AccessZone, IsiPath, IsiVolumePathPermissions, AzServiceIP.
```
Comment (rsedlock1958) [inline content/v3/csidriver/features/powerscale.md:348]: Change "Following" to "These"

File: content/v3/csidriver/features/powerscale.md [hunk 10]
```diff
@@ -440,7 +448,7 @@ The user can also set the volume limit for all the nodes in the cluster by speci
 
 ## Node selector in helm template
 
-Now user can define in which worker node, the CSI node pod daemonset can run (just like any other pod in Kubernetes world.)For more information, refer to https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector 
+Now user can define in which worker node, the CSI node pod daemonset can run (just like any other pod in Kubernetes world.)For more information, refer to https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
```
Comment (rsedlock1958) [inline content/v3/csidriver/features/powerscale.md:451]: Change "user" to "users". Remove comma after "node". Remove parenthethes.

File: content/v3/csidriver/features/powerscale.md [hunk 11]
```diff
@@ -624,14 +640,14 @@ In the case of operator-based installation, default permission for powerscale di
...
 ## PV/PVC Metrics
 
-CSI Driver for Dell PowerScale 2.1.0 and above supports volume health monitoring. This allows Kubernetes to report on the condition, status and usage of the underlying volumes. 
+CSI Driver for Dell PowerScale 2.1.0 and above supports volume health monitoring. This allows Kubernetes to report on the condition, status and usage of the underlying volumes.
 For example, if a volume were to be deleted from the array, or unmounted outside of Kubernetes, Kubernetes will now report these abnormal conditions as events.
 
 ### This feature can be enabled
-1. For controller plugin, by setting attribute `controller.healthMonitor.enabled` to `true` in `values.yaml` file. Also health monitoring interval can be changed through attribute `controller.healthMonitor.interval` in `values.yaml` file.   
+
+1. For controller plugin, by setting attribute `controller.healthMonitor.enabled` to `true` in `values.yaml` file. Also health monitoring interval can be changed through attribute `controller.healthMonitor.interval` in `values.yaml` file.
```
Comment (rsedlock1958) [inline content/v3/csidriver/features/powerscale.md:650]: First clause is not a complete sentence.

File: content/v1/csidriver/features/powerscale.md (hunk 1 above)
Comment (chimanjain) [reply content/v1/csidriver/features/powerscale.md:349]: Ok

File: content/v2/csidriver/features/powerscale.md (hunk 7 above)
Comment (chimanjain) [reply content/v2/csidriver/features/powerscale.md]: Done

File: content/v2/csidriver/features/powerscale.md (hunk 6 above)
Comment (chimanjain) [reply content/v2/csidriver/features/powerscale.md:356]: Done

File: content/v2/csidriver/features/powerscale.md (hunk 5 above)
Comment (chimanjain) [reply content/v2/csidriver/features/powerscale.md]: Done

**Review by shanmydell — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Pull request merged**